### PR TITLE
fix(dummy): add payload input to dummy app

### PR DESCRIPTION
**Pull request by fkm**

### CODE REVIEW

**Comment by fkm** [comment]
*bump*

File: tests/dummy/app/templates/application.hbs [hunk 1]
```diff
@@ -3,11 +3,25 @@
 {{manage-files}}
 
 {{!--
-{{manage-files/delete templates=templates}}
+{{manage-files/delete}}
 
-{{manage-files/upload templates=templates}}
+{{manage-files/upload}}
 --}}
```
Comment (anehx) [inline tests/dummy/app/templates/application.hbs]: What's the purpose of this comment?

**Review by anehx — APPROVED** [review]

File: tests/dummy/app/templates/application.hbs (hunk 1 above)
Comment (fkm) [reply tests/dummy/app/templates/application.hbs]: To show that the manage-files components can be used separately.

Comment (anehx) [reply tests/dummy/app/templates/application.hbs]: Hm then maybe uncomment it and add a descriptive title..

Comment (fkm) [reply tests/dummy/app/templates/application.hbs]: It's not in the code anymore.

**Pull request merged**